selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/HomePagePlatform.java

```diff
@@ -101,11 +101,8 @@ public void goToTaskPage() {
    */
   public void goToMySpaces() {
     info("-- Go to my spaces --");
-    ELEMENT_MY_SPACE_LINK_PLF.waitUntil(Condition.visible, Configuration.timeout);
-    sleep(2000);
-    ELEMENT_MY_SPACE_LINK_PLF.click();
-    sleep(2000);
-    refreshUntil(ELEMENT_ADDNEWSPACE_BUTTON, Condition.visible, 1000);
+    ELEMENT_MY_SPACE_LINK_PLF.waitUntil(Condition.visible,Configuration.collectionsTimeout).click();
+    refreshUntil(ELEMENT_ADDNEWSPACE_BUTTON, Condition.visible, 3000);
```

Comment:
Shouldn't we use `Configuration.timeout` instead of `3000` ?

Reply:
Yes :)